app-modules/campaign/src/Actions/ExecuteCampaignAction.php

```diff
@@ -63,6 +64,9 @@ public function uniqueId(): string
 
     public function handle(): void
     {
+        if (CancelCampaignAction::active() && $this->action->cancelled_at !== null) {
+            return;
+        }
```

Comment:
We should give code some room to breath in most scenarios. It makes it a lot easier to read.

```suggestion
        if (CancelCampaignAction::active() && $this->action->cancelled_at !== null) {
            return;
        }
        
```

Reply:
But when I run the format command it is removing the space